docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -451,12 +469,28 @@ The following table lists the measurement data that the HRM sensor provides.
 | Timestamp                   | `unsigned long long` | -                                   | Microseconds |
 | values[0]: Beats per minute | `int`                | Min. value = 0<br> Max. value = 240 | -            |
 
+<a name="hrm_batch"></a>
+## Heart Rate Monitor Batch Sensor
+
+The Heart Rate Monitor (HRM) batch sensor measures a person's heart rate. The sensor must be power-efficient and work in a batch manner.
```

Comment:
As a conclusion on our discussion https://github.com/Samsung/tizen-docs/pull/1083#discussion_r411830136 , we suggested the following update: 
```suggestion
The Heart Rate Monitor (HRM) batch sensor measures a person's heart rate. The sensor must be power-efficient and support batch processing.
```
@bbrto21 if you think this update doesn't change the meaning,
would you please apply this update?